desktop-widgets/mainwindow.cpp

```diff
@@ -194,7 +194,7 @@ MainWindow::MainWindow() : QMainWindow(),
 	registerApplicationState("PlanDive", divePlannerWidget, profileContainer, divePlannerSettingsWidget, plannerDetails );
 	registerApplicationState("EditPlannedDive", divePlannerWidget, profileContainer, diveList, mapWidget );
 	registerApplicationState("EditDiveSite", diveSiteEdit, profileContainer, diveList, mapWidget);
-	registerApplicationState("FilterDive", mainTab, profileContainer, diveListView, filterWidget2);
+	registerApplicationState("FilterDive", mainTab, profileContainer, diveList, filterWidget2);
```

Comment:
this, too, shouldn't be a separate commit and should just be done correctly in the first place. Otherwise the series fails when trying `git bisect` and hitting one of the earlier commits.